apps/aehyperchains/src/aehc_consensus_hyperchains.erl

```diff
@@ -905,14 +905,12 @@ verify_existing_staking_contract(Address, Trees, TxEnv) ->
                             OnchainAbi = aect_contracts:abi_version(Contract),
                             OnchainVm = aect_contracts:vm_version(Contract),
 
-                            %% TODO: Remove the stripping for IRIS...
-                            Code0 = aeser_contract_code:deserialize(get_staking_contract_bytecode()),
-                            FateCode0  = aeb_fate_code:deserialize(maps:get(byte_code, Code0)),
-                            FateCode1 = aeb_fate_code:strip_init_function(FateCode0),
-                            Bytecode = aeb_fate_code:serialize(FateCode1),
-                            LocalCode = aeser_contract_code:serialize(Code0#{ byte_code => Bytecode
-                                                                            , compiler_version => <<"unknown">>
-                                                                            }),
+                            Code = aeser_contract_code:deserialize(get_staking_contract_bytecode()),
+                            FateCode  = aeb_fate_code:deserialize(maps:get(byte_code, Code)),
+                            Bytecode = aeb_fate_code:serialize(FateCode),
```

Comment:
This is actually redundant. If you look closely you should see that:
```
maps:get(byte_code, Code) = Bytecode
```
Only think that may still be needed is setting the compiler version to "unknown"

Reply:
(deserialized just to serialize back is a no-op (or to be more precise an irrelevant assert on format))

Reply:
Thanks for pointing out.